users/serializers.py

```diff
@@ -34,6 +34,6 @@ def validate(self, user_data):
             raise serializers.ValidationError('Invalid Email or Password.')
             
         token = Token.objects.get_or_create(user=user)
```

Comment:
don't do this. assign the returned value to each variable.

`token, created = Token.objects...`